client/src/components/create-board-dialog.tsx

```diff
@@ -0,0 +1,64 @@
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogClose,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { Textarea } from "@/components/ui/textarea";
+
+export default function CreateBoardDialog() {
+  return (
+    <Dialog>
+      <DialogTrigger>
+        <Button>Create New Board</Button>
+      </DialogTrigger>
+      <DialogContent>
+        <DialogHeader>
+          <DialogTitle>Create New Board</DialogTitle>
+          <DialogDescription></DialogDescription>
+        </DialogHeader>
+        <div className="grid gap-4 py-4">
+          <div className="grid  items-center gap-4">
+            <Label htmlFor="name" className="text-left text-sm">
```

Comment:
I'm thinking we can remove this label since the dialog title is already "Create New Board"

Reply:
Removed! I think it's better that way.